### PR TITLE
JAX-WS: Testing soap response encoding

### LINKED ISSUE #27298: JAX-WS: Testing SOAP response encoding
This is the test for issue #27220

### PR DESCRIPTION
**Pull request by BerksanAtes**
Fixes #27298

### CODE REVIEW

**Comment by BerksanAtes** [comment]
#build

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/fat/src/com/ibm/ws/jaxws/fat/EncodingTest.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.HttpUtils;
+
+/**
+ * The purpose of this test is to check default encoding of
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/fat/src/com/ibm/ws/jaxws/fat/EncodingTest.java]:
```suggestion
 * The purpose of this test is to check the encoding we set in the Response's HTTP Headers
```

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/encodingTest/src/com/ibm/ws/jaxws/test/wsr/test/servlet/EncodingHttpTestServlet.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+import javax.xml.ws.handler.MessageContext;
+
+import com.ibm.ws.jaxws.test.wsr.server.stub.People;
+import com.ibm.ws.jaxws.test.wsr.server.stub.PeopleService;
+
+/**
+ *
+ */
+@WebServlet("/EncodingHttpTestServlet")
+public class EncodingHttpTestServlet extends HttpServlet {
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/encodingTest/src/com/ibm/ws/jaxws/test/wsr/test/servlet/EncodingHttpTestServlet.java]: To following a convetion we've been using, could we rename the servlet to just be `junit test class name + servlet`?

```suggestion
public class EncodingTestServlet extends HttpServlet {
```

**Review by neuwerk — CHANGES_REQUESTED** [review]
Just a few suggestions.

**Comment by BerksanAtes** [comment]
#build

**Review by neuwerk — APPROVED** [review]

**Comment by BerksanAtes** [comment]
#run-libby-bot

**Pull request merged**